input/community/speakers/index.cshtml

```diff
@@ -37,7 +37,7 @@ Title: Speaker Directory
                     <div class="form-group row">
                         <label class="col-sm-2 col-form-label">Filter By Topic</label>
                         <div class="col-sm-10" id="topic-filters">
-                            @foreach (string topic in OutputPages["community/speakers/*.html"].Where(x => !x.IdEquals(Document)).SelectMany(x => x.GetList<string>(SiteKeys.Topics)).Distinct().OrderBy(x => x))
+                            @foreach (string topic in OutputPages["community/speakers/*.html"].Where(x => !x.IdEquals(Document)).SelectMany(x => x.GetList<string>(SiteKeys.Topics, Array.Empty<string>())).Distinct().OrderBy(x => x))
```

Comment:
This threw if the speaker file didn't contain any topics since the default value for `.GetList(...)` is null when the key doesn't exist. Changed it to an empty result set so the `.SelectMany()` doesn't complain about getting a null result for one of the items (same with the language collection below).